chapters/synchronous.tex

```diff
@@ -127,30 +122,32 @@ \subsection{Rationale for Clocked Semantics}\label{rationale-for-clocked-semanti
     controller part, and different integer multiples are given, then the
     translator has to accept this (no error is detected).
   \end{enumerate}
+  Note: Clocked systems can mix different sampling rates
+  in well-defined ways when needed.
 \item
-  Due to the automatic sample and hold semantics, all variables
-  assigned in a when-clause of the above kind must have an initial value
+  Fewer initial conditions are needed, as only a subset of clocked
+  variables need initial conditions - the clocked state variables \cref{clocked-state-variables}.
+  For a standard when-clause all variables
+  assigned in a when-clause must have an initial value
   because they might be used, before they are assigned a value the first
   time. As a result, all these variables are ``discrete-time states''
```

Comment:
I think it would be good if we could avoid using the sarcasm-marked term _discrete-time states_.  Given the sentence before this one, can't we just skip the "discrete-time states" and say something like this:
> In reality, however, only the initialization of a subset of them makes a real difference to the simulation result.